mypy.ini

```diff
@@ -0,0 +1,44 @@
ipe_scrapers.marthastewart,recipe_scrapers.matprat,recipe_scrapers.meljoulwan,recipe_scrapers.melskitchencafe,recipe_scrapers.mindmegette,recipe_scrapers.minimalistbaker,recipe_scrapers.misya,recipe_scrapers.mobkitchen,recipe_scrapers.momswithcrockpots,recipe_scrapers.monsieurcuisine,recipe_scrapers.motherthyme,recipe_scrapers.mybakingaddiction,recipe_scrapers.mykitchen101,recipe_scrapers.mykitchen101en,recipe_scrapers.myrecipes,recipe_scrapers.nhshealthierfamilies,recipe_scrapers.nihhealthyeating,recipe_scrapers.nourishedbynutrition,recipe_scrapers.nutritionbynathalie,recipe_scrapers.nytimes,recipe_scrapers.ohsheglows,recipe_scrapers.onehundredonecookbooks,recipe_scrapers.owenhan,recipe_scrapers.paleorunningmomma,recipe_scrapers.panelinha,recipe_scrapers.paninihappy,recipe_scrapers.pingodoce,recipe_scrapers.plugins._interface,recipe_scrapers.plugins.exception_handling,recipe_scrapers.plugins.html_tags_stripper,recipe_scrapers.plugins.normalize_string,recipe_scrapers.plugins.opengraph_image_fetch,recipe_scrapers.plugins.schemaorg_fill,recipe_scrapers.plugins.template,recipe_scrapers.popsugar,recipe_scrapers.practicalselfreliance,recipe_scrapers.primaledgehealth,recipe_scrapers.przepisy,recipe_scrapers.purelypope,recipe_scrapers.purplecarrot,recipe_scrapers.rachlmansfield,recipe_scrapers.rainbowplantlife,recipe_scrapers.realfoodtesco,recipe_scrapers.realsimple,recipe_scrapers.recipetineats,recipe_scrapers.redhousespice,recipe_scrapers.reishunger,recipe_scrapers.rezeptwelt,recipe_scrapers.sallysbakingaddiction,recipe_scrapers.sallysblog,recipe_scrapers.saveur,recipe_scrapers.seriouseats,recipe_scrapers.settings.__init__,recipe_scrapers.simplyquinoa,recipe_scrapers.simplyrecipes,recipe_scrapers.simplywhisked,recipe_scrapers.skinnytaste,recipe_scrapers.southernliving,recipe_scrapers.spendwithpennies,recipe_scrapers.springlane,recipe_scrapers.steamykitchen,recipe_scrapers.streetkitchen,recipe_scrapers.sunbasket,recipe_scrapers.sundpaabudget,recipe_scrapers.sweetcsdesigns,recipe_scrapers.sweetpeasandsaffron,recipe_scrapers.tasteofhome,recipe_scrapers.tastesbetterfromscratch,recipe_scrapers.tastesoflizzyt,recipe_scrapers.tasty,recipe_scrapers.tastykitchen,recipe_scrapers.theclevercarrot,recipe_scrapers.thehappyfoodie,recipe_scrapers.thekitchenmagpie,recipe_scrapers.thekitchn,recipe_scrapers.thenutritiouskitchen,recipe_scrapers.thepioneerwoman,recipe_scrapers.thespruceeats,recipe_scrapers.thevintagemixer,recipe_scrapers.thewoksoflife,recipe_scrapers.timesofindia,recipe_scrapers.tineno,recipe_scrapers.tudogostoso,recipe_scrapers.twopeasandtheirpod,recipe_scrapers.usdamyplate,recipe_scrapers.valdemarsro,recipe_scrapers.vanillaandbean,recipe_scrapers.vegolosi,recipe_scrapers.vegrecipesofindia,recipe_scrapers.watchwhatueat,recipe_scrapers.whatsgabycooking,recipe_scrapers.wholefoods,recipe_scrapers.wikicookbook,recipe_scrapers.woolworths,recipe_scrapers.woop,recipe_scrapers.yemek,recipe_scrapers.yummly,recipe_scrapers.zeitwochenmarkt,recipe_scrapers.zenbelly]
```

Comment:
I'm sorry that you had to do this. If I understand this correctly, you are saying for all the mentioned modules un-typed definitions are okay?

Can we not glob this?

Reply:
wasn't hard to add these, `mypy . | nvim -` and some replace

we can do some level of globbing, but the idea here is future files / modules will be under stricter settings, then we can gradually remove these files from the ignore list.

Reply:
Is it possible to put each entry onto a separate line?  (so that we get easier-to-read line-based diffs when removing items from here)

Reply:
I think the two options are:
a. combine the file names w/ commas and use one block (the current setup)
b. have each filename have its own block `disallow_untyped_defs=False` -- this is longer file wise but doesn't suffer from the terrible diffs

Reply:
Makes sense... pesky `ini` file format limitations :/

After a bit more reading, there could be a third option, too: placing an inline `# mypy: disallow-untyped-defs` in each affected file.

Vague idea: perhaps we'd be more likely to remember to go back and clean-up the affected scrapers if we see that directive when we're reading/editing the scraper files (so that it reminds us that method typings are missing)?

(I don't have strong feelings about this but I'm trying to figure out what the cleanup and maintenance process will be like, and how to make that straightforward)

Reply:
Oh didn't know about that feature, I like it!

Thought it was going to be a pain to update all the files, but a little python made it easy:

```python
from pathlib import Path


files = """
recipe_scrapers/_abstract.py
recipe_scrapers/_exceptions.py
recipe_scrapers/_schemaorg.py
recipe_scrapers/_utils.py
recipe_scrapers/abril.py
recipe_scrapers/acouplecooks.py
recipe_scrapers/afghankitchenrecipes.py
recipe_scrapers/akispetretzikis.py
recipe_scrapers/albertheijn.py
recipe_scrapers/allrecipes.py
recipe_scrapers/alltomat.py
recipe_scrapers/altonbrown.py
recipe_scrapers/amazingribs.py
recipe_scrapers/ambitiouskitchen.py
recipe_scrapers/archanaskitchen.py
recipe_scrapers/arla.py
recipe_scrapers/atelierdeschefs.py
recipe_scrapers/averiecooks.py
recipe_scrapers/bakingmischeif.py
recipe_scrapers/bakingsense.py
recipe_scrapers/bbcfood.py
recipe_scrapers/bbcgoodfood.py
recipe_scrapers/bettybossi.py
recipe_scrapers/bettycrocker.py
recipe_scrapers/bigoven.py
recipe_scrapers/blueapron.py
recipe_scrapers/bodybuilding.py
recipe_scrapers/bonappetit.py
recipe_scrapers/bowlofdelicious.py
recipe_scrapers/budgetbytes.py
recipe_scrapers/castironketo.py
recipe_scrapers/cdkitchen.py
recipe_scrapers/chefkoch.py
recipe_scrapers/closetcooking.py
recipe_scrapers/comidinhasdochef.py
recipe_scrapers/cookeatshare.py
recipe_scrapers/cookieandkate.py
recipe_scrapers/cookingcircle.py
recipe_scrapers/cookinglight.py
recipe_scrapers/cookpad.py
recipe_scrapers/cookstr.py
recipe_scrapers/coop.py
recipe_scrapers/copykat.py
recipe_scrapers/countryliving.py
recipe_scrapers/cucchiaio.py
recipe_scrapers/cuisineaz.py
recipe_scrapers/cybercook.py
recipe_scrapers/davidlebovitz.py
recipe_scrapers/delish.py
recipe_scrapers/ditchthecarbs.py
recipe_scrapers/domesticateme.py
recipe_scrapers/downshiftology.py
recipe_scrapers/dr.py
recipe_scrapers/eatingbirdfood.py
recipe_scrapers/eatingwell.py
recipe_scrapers/eatsmarter.py
recipe_scrapers/eatwhattonight.py
recipe_scrapers/epicurious.py
recipe_scrapers/ethanchlebowski.py
recipe_scrapers/farmhousedelivery.py
recipe_scrapers/fifteenspatulas.py
recipe_scrapers/finedininglovers.py
recipe_scrapers/fitmencook.py
recipe_scrapers/food.py
recipe_scrapers/food52.py
recipe_scrapers/foodandwine.py
recipe_scrapers/foodnetwork.py
recipe_scrapers/foodrepublic.py
recipe_scrapers/forksoverknives.py
recipe_scrapers/franzoesischkochen.py
recipe_scrapers/fredriksfikaallas.py
recipe_scrapers/g750g.py
recipe_scrapers/geniuskitchen.py
recipe_scrapers/giallozafferano.py
recipe_scrapers/gimmesomeoven.py
recipe_scrapers/globo.py
recipe_scrapers/gonnawantseconds.py
recipe_scrapers/gousto.py
recipe_scrapers/goustojson.py
recipe_scrapers/greatbritishchefs.py
recipe_scrapers/halfbakedharvest.py
recipe_scrapers/hassenchef.py
recipe_scrapers/headbangerskitchen.py
recipe_scrapers/heb.py
recipe_scrapers/heinzbrasil.py
recipe_scrapers/hellofresh.py
recipe_scrapers/homechef.py
recipe_scrapers/hostthetoast.py
recipe_scrapers/hundredandonecookbooks.py
recipe_scrapers/ica.py
recipe_scrapers/ig.py
recipe_scrapers/imworthy.py
recipe_scrapers/indianhealthyrecipes.py
recipe_scrapers/innit.py
recipe_scrapers/inspiralized.py
recipe_scrapers/jamieoliver.py
recipe_scrapers/jimcooksfoodgood.py
recipe_scrapers/joyfoodsunshine.py
recipe_scrapers/justataste.py
recipe_scrapers/justbento.py
recipe_scrapers/justonecookbook.py
recipe_scrapers/kennymcgovern.py
recipe_scrapers/kingarthur.py
recipe_scrapers/kitchenstories.py
recipe_scrapers/kochbar.py
recipe_scrapers/koket.py
recipe_scrapers/kptncook.py
recipe_scrapers/kuchniadomowa.py
recipe_scrapers/kwestiasmaku.py
recipe_scrapers/latelierderoxane.py
recipe_scrapers/lecker.py
recipe_scrapers/lecremedelacrumb.py
recipe_scrapers/lekkerensimpel.py
recipe_scrapers/littlespicejar.py
recipe_scrapers/livelytable.py
recipe_scrapers/lovingitvegan.py
recipe_scrapers/madensverden.py
recipe_scrapers/madewithlau.py
recipe_scrapers/marleyspoon.py
recipe_scrapers/marmiton.py
recipe_scrapers/marthastewart.py
recipe_scrapers/matprat.py
recipe_scrapers/meljoulwan.py
recipe_scrapers/melskitchencafe.py
recipe_scrapers/mindmegette.py
recipe_scrapers/minimalistbaker.py
recipe_scrapers/misya.py
recipe_scrapers/mobkitchen.py
recipe_scrapers/momswithcrockpots.py
recipe_scrapers/monsieurcuisine.py
recipe_scrapers/motherthyme.py
recipe_scrapers/mybakingaddiction.py
recipe_scrapers/mykitchen101.py
recipe_scrapers/mykitchen101en.py
recipe_scrapers/myrecipes.py
recipe_scrapers/nhshealthierfamilies.py
recipe_scrapers/nihhealthyeating.py
recipe_scrapers/nourishedbynutrition.py
recipe_scrapers/nutritionbynathalie.py
recipe_scrapers/nytimes.py
recipe_scrapers/ohsheglows.py
recipe_scrapers/onehundredonecookbooks.py
recipe_scrapers/owenhan.py
recipe_scrapers/paleorunningmomma.py
recipe_scrapers/panelinha.py
recipe_scrapers/paninihappy.py
recipe_scrapers/pingodoce.py
recipe_scrapers/plugins/_interface.py
recipe_scrapers/plugins/exception_handling.py
recipe_scrapers/plugins/html_tags_stripper.py
recipe_scrapers/plugins/normalize_string.py
recipe_scrapers/plugins/opengraph_image_fetch.py
recipe_scrapers/plugins/schemaorg_fill.py
recipe_scrapers/plugins/template.py
recipe_scrapers/popsugar.py
recipe_scrapers/practicalselfreliance.py
recipe_scrapers/primaledgehealth.py
recipe_scrapers/przepisy.py
recipe_scrapers/purelypope.py
recipe_scrapers/purplecarrot.py
recipe_scrapers/rachlmansfield.py
recipe_scrapers/rainbowplantlife.py
recipe_scrapers/realfoodtesco.py
recipe_scrapers/realsimple.py
recipe_scrapers/recipetineats.py
recipe_scrapers/redhousespice.py
recipe_scrapers/reishunger.py
recipe_scrapers/rezeptwelt.py
recipe_scrapers/sallysbakingaddiction.py
recipe_scrapers/sallysblog.py
recipe_scrapers/saveur.py
recipe_scrapers/seriouseats.py
recipe_scrapers/simplyquinoa.py
recipe_scrapers/simplyrecipes.py
recipe_scrapers/simplywhisked.py
recipe_scrapers/skinnytaste.py
recipe_scrapers/southernliving.py
recipe_scrapers/spendwithpennies.py
recipe_scrapers/springlane.py
recipe_scrapers/steamykitchen.py
recipe_scrapers/streetkitchen.py
recipe_scrapers/sunbasket.py
recipe_scrapers/sundpaabudget.py
recipe_scrapers/sweetcsdesigns.py
recipe_scrapers/sweetpeasandsaffron.py
recipe_scrapers/tasteofhome.py
recipe_scrapers/tastesbetterfromscratch.py
recipe_scrapers/tastesoflizzyt.py
recipe_scrapers/tasty.py
recipe_scrapers/tastykitchen.py
recipe_scrapers/theclevercarrot.py
recipe_scrapers/thehappyfoodie.py
recipe_scrapers/thekitchenmagpie.py
recipe_scrapers/thekitchn.py
recipe_scrapers/thenutritiouskitchen.py
recipe_scrapers/thepioneerwoman.py
recipe_scrapers/thespruceeats.py
recipe_scrapers/thevintagemixer.py
recipe_scrapers/thewoksoflife.py
recipe_scrapers/timesofindia.py
recipe_scrapers/tineno.py
recipe_scrapers/tudogostoso.py
recipe_scrapers/twopeasandtheirpod.py
recipe_scrapers/usdamyplate.py
recipe_scrapers/valdemarsro.py
recipe_scrapers/vanillaandbean.py
recipe_scrapers/vegolosi.py
recipe_scrapers/vegrecipesofindia.py
recipe_scrapers/watchwhatueat.py
recipe_scrapers/whatsgabycooking.py
recipe_scrapers/wholefoods.py
recipe_scrapers/wikicookbook.py
recipe_scrapers/woolworths.py
recipe_scrapers/woop.py
recipe_scrapers/yemek.py
recipe_scrapers/yummly.py
recipe_scrapers/zeitwochenmarkt.py
recipe_scrapers/zenbelly.py
"""

for file in files.strip().splitlines():
    print(f"updating {file}")
    text = (Path(".") / file).read_text()
    text = "# mypy: disallow_untyped_defs=False\n" + text
    Path(file).write_text(text)
```

Reply:
Nicely done :)  Thank you!